lib/suspenders/adapters/heroku.rb

```diff
@@ -0,0 +1,52 @@
+module Suspenders
+  module Adapters
+    class Heroku
+      def initialize(app_builder)
+        @app_builder = app_builder
+      end
+
+      def set_heroku_remotes
+        remotes = <<-SHELL.strip_heredoc
+
+          # Set up the staging and production apps.
+          #{join_heroku_app('staging')}
+          #{join_heroku_app('production')}
+        SHELL
+
+        @app_builder.append_file "bin/setup", remotes
+      end
+
+      private
+
+      def join_heroku_app(environment)
+        heroku_app_name = heroku_app_name_for(environment)
+        <<-SHELL.strip_heredoc
+          if heroku join --app #{heroku_app_name} &> /dev/null; then
+            git remote add #{environment} git@heroku.com:#{heroku_app_name}.git || true
+            printf 'You are a collaborator on the "#{heroku_app_name}" Heroku app\n'
```

Comment:
I think it's much uglier looking, but I have no really good reason not to do that. Will do.